### DISCUSSION TITLE
Show error when using date pickers (start date greater than end date)

### DISCUSSION THREAD

**Discussion by lemonpeu** [react-hook-form/react-hook-form#3184] [Q&A] [answered]
Hello, sorry if my English is not the best. I have this piece of code and I want to show an error when the user selects a start date greater than the end date
Just in case I'm using ant design for the date picker
```
<Controller
            render={(props) => (
              <DatePicker
                disabledDate={disabledDate}
                picker="month"
                {...props}
                onChange={(e) => {
                  props.onChange(e);
                }}
              />
            )}
            control={control}
            name="issued"
            defaultValue={editMode ? moment(editData.issued) : null}
            rules={{ required: true }}
          />

{errors.issued && watchStartDate > watchEndDate ? 'Start date shouldn't be greater than end date' : ''}

```
watchStartDate watches the input value inside the startdate date picker

Thanks in advance ;)

**Comment by bluebill1049** [comment, marked as answer]
can you share a codesandbox?

**Reply by kaisiant** [reply]
> Hello, sorry if my English is not the best. I have this piece of code and I want to show an error when the user selects a start date greater than the end date
> Just in case I'm using ant design for the date picker
> 
> ```
> <Controller
>             render={(props) => (
>               <DatePicker
>                 disabledDate={disabledDate}
>                 picker="month"
>                 {...props}
>                 onChange={(e) => {
>                   props.onChange(e);
>                 }}
>               />
>             )}
>             control={control}
>             name="issued"
>             defaultValue={editMode ? moment(editData.issued) : null}
>             rules={{ required: true }}
>           />
> 
> {errors.issued && watchStartDate > watchEndDate ? 'Start date shouldn't be greater than end date' : ''}
> ```
> 
> watchStartDate watches the input value inside the startdate date picker
> 
> Thanks in advance ;)

Here you go https://codesandbox.io/s/react-hook-form-v6-controller-forked-2k6jt?file=/src/AntD.js

**Reply by bluebill1049** [reply]
ty @kaisian97
